NOTICE

```diff
@@ -0,0 +1,21 @@
+# Notice
+
+This project contains several dependencies which have been vendored in
+due to a need for minor changes. Where possible changes have been
+contributed back to the original project.
+
+## async-listener
+
+- *path:* lib/instrumentation/patch-async
+- *author:* Forrest L Norvell
+- *project url:* https://github.com/othiym23/async-listener
+- *original file:* https://github.com/othiym23/async-listener/blob/master/index.js
+- *license:* BSD-2-Clause, http://opensource.org/licenses/BSD-2-Clause
+
+## shimmer
+
+- *path:* lib/instrumentation/shimmer
+- *author:* Forrest L Norvell
+- *project url:* https://github.com/othiym23/shimmer
+- *original file:* https://github.com/othiym23/shimmer/blob/master/index.js
+- *license:* BSD-2-Clause, http://opensource.org/licenses/BSD-2-Clause
```

Comment:
Below here, we need to add the following in order to comply with the BSD-2 license:

```
Copyright (c) 2013-2019, Forrest L Norvell
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are met:

* Redistributions of source code must retain the above copyright notice, this
  list of conditions and the following disclaimer.

* Redistributions in binary form must reproduce the above copyright notice,
  this list of conditions and the following disclaimer in the documentation
  and/or other materials provided with the distribution.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```